docs/contributing/platform-api/common.rst

```diff
@@ -1,11 +1,5 @@
-pulp.common
-===============
+pulp.constants
```

Comment:
Should this file be renamed to constants.rst?

Reply:
+1 renaming to constants.rst for consistency